include/swift/IDE/Formatting.h

```diff
@@ -20,8 +20,8 @@ struct CodeFormatOptions {
 public:
   bool UseTabs = false;
   bool IndentSwitchCase = false;
-  unsigned IndentWidth = 4;
-  unsigned TabWidth = 4;
+  unsigned IndentWidth = 2;
+  unsigned TabWidth = 8;
```

Comment:
It's not a problem for Xcode. I just think the defaults for the command line tool should match the defaults in Xcode.